scripts/debug.py

```diff
@@ -0,0 +1,15 @@
+from NBA.models import *
+from nba_api.stats.static import teams,players
+from nba_api.stats.endpoints import leaguegamefinder,playergamelog,playergamelogs
+import time
+import pandas as pd
+from NBA.update import Update
+from nba_api.stats.library.parameters import SeasonAll
+from NBA.nba_api_helper import NBA_ApiHelper
+from . import InitialCompute
+
+def run():
```

Comment:
What is the purpose of this if it just prints `hello`? 

What was the intention?